mintapi/signIn.py

```diff
@@ -295,7 +317,8 @@ def _create_web_driver_at_mint_com(
         # chrome_options.add_argument("--window-size=1920x1080")
     if session_path is not None:
         chrome_options.add_argument("user-data-dir=%s" % session_path)
-
+    chrome_options.add_experimental_option("useAutomationExtension", False)
```

Comment:
What do these add in this branch?

Reply:
It assisted me in the CAPTCHA part. I will remove it